MAINTAINERS.md

```diff
@@ -0,0 +1,20 @@
+The current maintainers of this repo are [@adamralph](https://github.com/adamralph), [@distantcam](https://github.com/distantcam), and [@HEskandari](https://github.com/HEskandari).
+
+The maintainers [watch](https://github.com/Particular/ServiceInsight/watchers) this repo and undertake the following responsibilities:
+
+- Ensuring that the `master` branch is always releasable. "Releasable" means that the software built from the latest commit can be released immediately simply by following the [release procedure](https://github.com/Particular/ServiceInsight/blob/master/docs/releasing.md).
```

Comment:
If the build hasn't run, and all the tests pass, then we can't release, so that is implicit.

I think some level of smoke testing should be done before merging PRs, based on judgement. I'll add a point about that to the merging PRs section.